ri_public_examples/download_files.py

```diff
@@ -0,0 +1,79 @@
+import os
+import argparse
+import subprocess
+from typing import List, Optional, Tuple
+from pathlib import Path
+
+
+GITHUB_URL = "https://raw.githubusercontent.com/RobustIntelligence/ri-public-examples/master/"
```

Comment:
you mean as data files? not sure what the benefits of that are vs. just hosting the links here (we'd still have to download all the raw data/model files somehow), what are your thoughts?

Reply:
wouldnt have to do the whole GITHUB_URL download and stuff (since it will already be in the package) so could just load from that file and not have to download

Reply:
mm it's just a bit more annoying because i have to add a manifest file + use importlib ( http://sixty-north.com/blog/including-package-data-in-python-packages.html), but ok i can try

Reply:
can also leave it as a todo?